src/kubernetes-ingress-controller/references/custom-resources.md

```diff
@@ -9,10 +9,10 @@ Following CRDs enables users to declaratively configure all aspects of Kong:
 
 - [**KongPlugin**](#kongplugin): This resource corresponds to
   the [Plugin][kong-plugin] entity in Kong.
-- [**KongIngress**](#kongingress): This resource provides fine-grained control
-  over all aspects of proxy behaviour like routing, load-balancing,
-  and health checking. It serves as an "extension" to the Ingress resources
-  in Kubernetes.
+- [**KongIngress**](#kongingress): {% if_version gte:2.8.x -%}**(Partially deprecated)**{% endif_version %}
+  This resource provides fine-grained control over all aspects of proxy
```

Comment:
```suggestion
- [**KongIngress**](#kongingress): {% if_version gte:2.8.x -%}**(Partially deprecated)**{% endif_version %}This resource provides fine-grained control over all aspects of proxy
```
Hoping this doesn't mess with the conditional rendering, just because the sentence is currently appearing on a new line instead of after the colon.